pkg/provider/k8s/k8s.go

```diff
@@ -179,6 +180,10 @@ func (c *K8s) ResourceApply(deployments []Resource) error {
 				err = c.customResourceApply(resource)
 			case "statefulset":
 				err = c.statefulSetApply(resource)
+			case "job":
+				err = c.jobApply(resource)
+				// Ignore the error as even when job deletion fails, it will be removed together with nodepool.
+				_ = c.jobDelete(resource)
```

Comment:
Why is this delete necessary?

Reply:
It's inherited from #311, at that time, we didn't have the step `make funcbench_resource_delete`. Now we can keep previous step and remove this line.